agagd/agagd_core/tables/beta.py

```diff
@@ -0,0 +1,403 @@
+# AGAGD Core Imports
+from agagd_core.models import Chapters
+from agagd_core.models import Game
+from agagd_core.models import Member
+from agagd_core.models import Tournament
+from agagd_core.models import TopDan
+from agagd_core.models import TopKyu
+from agagd_core.models import MostTournamentsPastYear
+from agagd_core.models import MostRatedGamesPastYear
+
+# Django Imports
+import django_tables2 as tables
```

Comment:
Would it make sense to have `/beta` omit `django_tables2`?

Reply:
Yes it does. I just am sort of avoiding this part right now.